paging/paging-common/src/test/kotlin/androidx/paging/PagingDataDifferTest.kt

```diff
@@ -1611,7 +1924,9 @@ private fun infinitelySuspendingPagingData(receiver: UiReceiver = dummyReceiver)
     receiver
 )
 
-private class UiReceiverFake : UiReceiver {
+private class UiReceiverFake(
```

Comment:
nit: If we're actually passing through to a real UiReceiver, I don't think this can be considered a fake anymore.

suggest: TrackableUiReceiverWrapper

Reply:
Done.